Classes/Emogrifier.php

```diff
@@ -394,6 +395,7 @@ protected function process(\DOMDocument $xmlDocument)
             $this->removeInvisibleNodes($xPath);
         }
 
+	$this->copyImportsToStyleNode($xmlDocument, $xPath, $cssImports);
```

Comment:
Please use 4 spaces for indentation, not tabs or 8 spaces. You can find the PSR-2 style guide here:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md

I highly recommend that you read it. It's not very long, and it's used by a lot of projects.